canonical-web-and-design/canonical.com
# Add and run inclusive naming action

## Done

- Adds inclusive naming action
- Runs the action to remove previously used non-inclusive language
- 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the github action works.
- Check the language changes make lexical sense.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4446
